src/commands/send_stream.rs

```diff
@@ -0,0 +1,99 @@
+use crate::action::{Action, Processing};
+use crate::commands::{Command};
+use crate::state::{State};
+use crate::message::{NetMessage};
+use crate::util::{Result, Reportable};
+
+use message_io::network::{Network};
+use v4l::prelude::*;
+use v4l::FourCC;
+
+// Send Stream logic
+
+pub struct SendStreamCommand;
+impl Command for SendStreamCommand {
+    fn name(&self) -> &'static str {
+        "stream"
+    }
+
+    fn parse_params(&self, _params: &[&str]) -> Result<Box<dyn Action>> {
+        match SendStream::new() {
```

Comment:
Check here if the OS is Linux,Windows,MacOS...

Reply:
Why, with the current cfg usage the new commands are only parsed in linux, on the other platform its considered a normal string.

Reply:
I guess you mean to add this with the change you requested above which makes sense.

Reply:
Yeah, this is referring to the above change, sorry :)